### PR TITLE
refactored to use the integration logic

### PR DESCRIPTION
**Pull request by bsy**
Refactor using the `@mavengi/knowledge-base-integration` package.

### CODE REVIEW

**Review by bsadecki-magi — APPROVED** [review]
Looks good to me. 🚢

File: src/index.ts [hunk 1]
```diff
@@ -22,108 +26,65 @@ async function callElevioApi(path: string, key: string, token: string) {
...
-      settings.key,
-      settings.token,
-      knowledgeBaseId.referenceId
-    );
+    console.log('Finished processing all articles');
   },
+  convertDataToMD: (documentBodyPayload: string) => {
+    const turndownService = new TurndownService();
+    turndownService.use(gfm);
+    return turndownService.turndown(documentBodyPayload);
```
Comment (cassiedoll) [inline src/index.ts:82]: you don't need turndown here. the api accepts html.
would prefer to pull this logic out of clients and keep it in platform - with the goal of clients being as simple as possible

File: .npmrc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@mavenagi:registry=https://registry.npmjs.org/
```
Comment (cassiedoll) [inline .npmrc]: do we need this? 
can the shared package just be public?

this repo is open so ideally this uses all open code.

File: instructions.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Product Requirements Document
```
Comment (cassiedoll) [inline instructions.md]: prob dont check in

File: .npmrc (hunk 2 above)
Comment (cassiedoll) [reply .npmrc]: this is why the vercel deployment failed btw

File: src/index.ts (hunk 1 above)
Comment (bsy) [reply src/index.ts:82]: Igor was saying it's better to ship in MD

File: .npmrc (hunk 2 above)
Comment (bsy) [reply .npmrc]: Yes I pinged @bsadecki-magi about this, I think we can just make the dependency public now

File: src/index.ts (hunk 1 above)
Comment (bsy) [reply src/index.ts:82]: @igorien2k was working on some logic to also convert all sorts of formats into MD

File: .npmrc (hunk 2 above)
Comment (bsadecki-magi) [reply .npmrc]: We're public now 🥳

File: src/index.ts (hunk 1 above)
Comment (bsadecki-magi) [reply src/index.ts:82]: Just published v1.0.0 of the library (contains breaking changes).

https://www.npmjs.com/package/@mavenagi/knowledge-base-integration

This allows you to choose `HTML|MD` content. Defaults to HTML and passthrough with examples in the README.

Comment (cassiedoll) [reply src/index.ts:82]: platform wise this is a very different code path than what igor was working with - cause this isn't a crawl. 
we basically do turndown for you on the backend - no llms involved here

Comment (cassiedoll) [reply src/index.ts:82]: oh and if you run into a different content type that is not html or markdown - definitely let us know! im sure we can find a library to parse it and turn it into markdown in the backend. 

the llm is only needed to clean up content which is nondeterministically messy - and generally speaking that shouldn't be necessary with any of these connectors as they are all first class apis. only crawls are a disaster really

**Pull request merged**